### PR TITLE
:book: CONTRIBUTING.md: Clarify contribution ladder

### PR DESCRIPTION
**Pull request by randomvariable**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,3 +17,60 @@ Kubernetes projects require that you sign a Contributor License Agreement (CLA)
...
+If you have made significant contributions to Cluster API Provider vSphere, a maintainer may nominate you to become a maintainer for the project.
+
+We generally follow the [requirements for a approver](https://github.com/kubernetes/community/blob/master/community-membership.md#approver) from upstream Kubernetes. However, if you don't fully meet the requirements then a quorum of maintainers may still propose you if they feel you will make significant contributions.
+
+Maintainers are able to approve PRs, as well as participate in release processes and have write access to the repo. **As a maintainer you will be expected to run the office hours, especially if no else wants to**.
+
+Maintainers require membership of the **Kubernetes** Github organisation via
+[kubernetes/org/issues/new/choose](https://github.com/kubernetes/org/issues/new/choose)
+
+The steps to add someone as a reviewer are:
```
Comment (zhanggbj) [inline CONTRIBUTING.md:56]: Perhaps a typo here? reviewer->maintainer

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,3 +17,60 @@ Kubernetes projects require that you sign a Contributor License Agreement (CLA)
...
+
+We broadly follow the requirements from the [Kubernetes Community Membership](https://github.com/kubernetes/community/blob/master/community-membership.md).
+
+> When making changes to **OWNER_ALIASES** please check that the **sig-cluster-lifecycle-leads**, **cluster-api-admins** and **cluster-api-maintainers** are correct.
+
+#### Becoming a reviewer
+
+If you would like to become a reviewer, then please ask one of the current maintainers.
+
+We generally try to follow the [requirements for a reviewer](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer) from upstream Kubernetes. But if you feel that you don't full meet the requirements then reach out to us, they are not set in stone.
```
Comment (sbueringer) [inline CONTRIBUTING.md:31]:
```suggestion
We generally try to follow the [requirements for a reviewer](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer) from upstream Kubernetes. But if you feel that you don't fully meet the requirements then reach out to us, they are not set in stone.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -17,3 +17,60 @@ Kubernetes projects require that you sign a Contributor License Agreement (CLA)
...
+* Announce the change within the CAPV slack channel and as a PSA in the next CAPV office hours
+* After 7 days of lazy consensus or after the next CAPV office hours (whichever is longer) the PR can be merged
+
+#### Becoming a maintainer
+
+If you have made significant contributions to Cluster API Provider vSphere, a maintainer may nominate you to become a maintainer for the project.
+
+We generally follow the [requirements for a approver](https://github.com/kubernetes/community/blob/master/community-membership.md#approver) from upstream Kubernetes. However, if you don't fully meet the requirements then a quorum of maintainers may still propose you if they feel you will make significant contributions.
+
+Maintainers are able to approve PRs, as well as participate in release processes and have write access to the repo. **As a maintainer you will be expected to run the office hours, especially if no else wants to**.
```
Comment (sbueringer) [inline CONTRIBUTING.md:51]:
```suggestion
Maintainers are able to approve PRs, as well as participate in release processes and have write access to the repo. **As a maintainer you will be expected to run the office hours, especially if no one else wants to**.
```

Nobody told me this one :D

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -17,3 +17,60 @@ Kubernetes projects require that you sign a Contributor License Agreement (CLA)
...
+
+Maintainers and require membership of the **Kubernetes** Github organisation via
```
Comment (sbueringer) [inline CONTRIBUTING.md:53]:
```suggestion
Maintainers require membership of the **Kubernetes** Github organisation via
```

**Review by sbueringer — COMMENTED** [review]
Just a few nits.

Overall lgtm

Thank you very much for writing this up, very clear

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -17,3 +17,61 @@ Kubernetes projects require that you sign a Contributor License Agreement (CLA)
...
+* Announce the change within the CAPV slack channel and as a PSA in the next CAPV office hours
+* After 7 days of lazy consensus or after the next CAPV office hours (whichever is longer) the PR can be merged
+* Open PR to add Github username to **cluster-api-provider-vsphere-maintainers**
+to [kubernetes/org/config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml)
+* Open PR to add Github username to [kubernetes/test-infra/config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/OWNERS](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/OWNERS)
+* Open PR to add Google ID to the k8s-infra-staging-capi-vsphere@kubernetes.io and sig-cluster-lifecycle-cluster-api-vsphere-alerts@kubernetes.io Google groups in [kubernetes/k8s.io/groups/sig-cluster-lifecycle/groups.yaml](https://github.com/kubernetes/k8s.io/blob/main/groups/sig-cluster-lifecycle/groups.yaml)
+* Open PR to image-builder to modify `cluster-api-vsphere-maintainers` in [OWNERS_ALIASES](https://github.com/kubernetes-sigs/image-builder/blob/main/OWNERS_ALIASES)
```
Comment (sbueringer) [inline CONTRIBUTING.md:66]: One more new place to update: https://github.com/kubernetes/k8s.io/pull/5642/files

Comment (randomvariable) [reply CONTRIBUTING.md:67]: Done

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Pull request merged**